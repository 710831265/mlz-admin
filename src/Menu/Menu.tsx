import React from 'react';
import { Menu as AntdMenu } from 'antd';

const Menu = () => {
  return AntdMenu;
};
export default Menu;
