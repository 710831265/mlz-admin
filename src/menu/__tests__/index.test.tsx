import React, { Component } from 'react';
import { testMount, testSnapshot } from '../../../tests';
import Menu from '..';

describe('ðŸ§ª Menu', () => {
  testMount(Menu);
  testSnapshot(Menu);
});
