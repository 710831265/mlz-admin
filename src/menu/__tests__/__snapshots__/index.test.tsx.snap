// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ðŸ§ª Menu snapshot test snapshot comparing 1`] = `"<ul class=\\"ant-menu ant-menu-light ant-menu-root ant-menu-vertical\\" role=\\"menu\\"></ul>"`;
